cfme/configure/access_control/__init__.py

```diff
@@ -700,7 +700,11 @@ def set_group_order(self, updated_order):
         Args:
             updated_order: group order list
         """
-        name_column = "Name"
+        if self.appliance.version < "5.9.2":
```

Comment:
**Optional**

Express as ternary, might be over 100 chars at this indent though.

```python
name_column = 'Name' if self.appliance.version < '5.9.2' else 'Description'
```